### PR TITLE
TEST: kill test

### PR DESCRIPTION
**Pull request by saltyan007**
This test for kill board driver tests all the buttons, computer and remote.

### CODE REVIEW

**Comment by RustyBamboo** [comment]
Should move these files to a new 'test' subdirectory in the navigator_kill_board package. Also, make sure your commit names follow the general convention i.e. `TOPIC: general what I changed here.`

**Comment by ironmig** [comment]
Any progress on this? I'll probably have someone finish the remaining issues next week if you don't so we can get it in our code

**Comment by saltyan007** [comment]
Finished reformation and comments adding, also create a new subdirectory 'test'. Now trying to push them on github. Should be finished by this week's meeting

**Comment by saltyan007** [comment]
Progress:
1. Reformation of the code with autopep8
2. Added comments
3. Moved all the test files to a new subdirectory "test"

**Comment by saltyan007** [comment]
New changes made:
1.deleted the files in directory 'nodes'
2.undo the change with irrelevant file

File: hardware_drivers/navigator_kill_board/test/kill_board_test.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+    def test_AG_remote(self):
+        self.assertEqual(self.AlarmListener.is_cleared(), True,
+                         msg='current state of kill signal is raised')
+        # publishing msg to network
+        pub = rospy.Publisher('/network', Header, queue_size=10)
+        rate = rospy.Rate(10)
+        t_end = rospy.Time.now() + rospy.Duration(3)
+        while rospy.Time.now() < t_end:
+            hello_header = Header()
```
Comment (RustyBamboo) [inline hardware_drivers/navigator_kill_board/test/kill_board_test.py]: hello!

**Comment by saltyan007** [comment]
CMakeLists.txt file added

**Comment by ironmig** [comment]
Nope. Those lines need to be added to the CMakeLisf that is already in that project

**Comment by saltyan007** [comment]
Fixed the problem u just mentioned

File: hardware_drivers/navigator_kill_board/CMakeLists.txt [hunk 2]
```diff
@@ -9,4 +9,8 @@ catkin_python_setup()
 
 catkin_package(
 )
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+  add_rostest(kill_board.test)
```
Comment (ironmig) [inline hardware_drivers/navigator_kill_board/CMakeLists.txt]: needs to be ```test/kill_board.test```. You NEED to be able to successfully run
```catkin_make run_tests_navigator_kill_board```, as this is what the CI does.

**Comment by saltyan007** [comment]
Should be all right this time, command catkin_make run_testes_navigator_kill_board worked this time.

**Comment by RustyBamboo** [comment]
@ironmig , @saltyan007 , seems good to me. I'd squash the commits when we merge

**Pull request merged**